src/main/java/seedu/address/logic/parser/DisplayCommandParser.java

```diff
@@ -20,14 +23,62 @@
     public DisplayCommand parse(String args) throws ParseException {
         requireNonNull(args);
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args);
+        List<String> argList = argMultimap.getAllValues(EMPTY_PREFIX);
+        String lastArgument = extractLastArgument(argList);
 
         Index index;
         try {
+            if (hasParameterValue(argList)) {
+                if (isWholeNumber(lastArgument)) {
+                    Integer.parseInt(lastArgument);
+                }
+            }
             index = ParserUtil.parseIndex(argMultimap.getPreamble());
         } catch (IllegalValueException ive) {
```

Comment:
what throws IllegalValueException? I don't seem to see anything that throws this. Is this block used?

Reply:
like if the input is 'display goat' then i think this is the exception that is thrown